crypto/ec/curve448/arch_ref64/arch_intrinsics.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#ifndef __ARCH_REF64_ARCH_INTRINSICS_H__
+# define __ARCH_REF64_ARCH_INTRINSICS_H__
+
+# define ARCH_WORD_BITS 64
+
+static __inline__ __attribute((always_inline, unused))
+uint64_t word_is_zero(uint64_t a)
+{
+    /* let's hope the compiler isn't clever enough to optimize this. */
```

Comment:
Don't we have a constant_time-function for that?

Reply:
Yes, and I've already pointed that out in the previous pull request

Reply:
I deleted this file, although this function appears elsewhere so I guess the same comment applies. As @kroeckx mentioned he previously raised this. My response at that time was:

> The types aren't quite the same, so this is probably a non-trivial exercise.

i.e. it is a non-trivial exercise to merge them (really they are different functions because of the different types)